wandb/sdk/wandb_setup.py

```diff
@@ -142,24 +140,27 @@ def settings(self, __d=None, **kwargs):
         s.update(__d, **kwargs)
         return s
 
+    def _get_entity(self):
+        entity = self._server._viewer.get("entity")
+        return entity
+
     def _load_viewer(self):
-        self._api = InternalApi()
-        self.mode = "run"
-        # TODO(jhr): Move this to settings
-        http_timeout = 5
-        if self.mode != "dryrun" and not self._api.disabled() and self._api.api_key:
-            # Kaggle has internet disabled by default, this checks for that case
-            async_viewer = util.async_call(self._api.viewer, timeout=http_timeout)
-            viewer, viewer_thread = async_viewer()
-            if viewer_thread.is_alive():
-                if _is_kaggle():
-                    raise CommError(
-                        "To use W&B in kaggle you must enable internet in the settings panel on the right."  # noqa: E501
-                    )
-            else:
-                # self._viewer = viewer
-                self._flags = json.loads(viewer.get("flags", "{}"))
-                # print("loadviewer3", self._flags, viewer)
+        s = server.Server()
+        s.query_with_timeout()
+        self._server = s
+        # if self.mode != "dryrun" and not self._api.disabled() and self._api.api_key:
+        #    # Kaggle has internet disabled by default, this checks for that case
+        #    async_viewer = util.async_call(self._api.viewer, timeout=http_timeout)
+        #    viewer, viewer_thread = async_viewer()
+        #    if viewer_thread.is_alive():
+        #        if _is_kaggle():
+        #            raise CommError(
+        #                "To use W&B in kaggle you must enable internet in the settings panel on the right."  # noqa: E501
+        #            )
+        #    else:
+        #        # self._viewer = viewer
+        #        self._flags = json.loads(viewer.get("flags", "{}"))
+        #        print("loadviewer3", self._flags, viewer)
```

Comment:
Is this going to come back in a later PR?

Reply:
yes, it is a reminder that it needs to be somewhere.  plan is:
1. wandb sync changes
2. offline mode
3. this code revived

This code relied on offline mode working, and there are some details that need to be flushed out before